src/base/bittorrent/session.cpp

```diff
@@ -4002,7 +4041,7 @@ void Session::handleTorrentRemovedAlert(const lt::torrent_removed_alert *p)
 
     if (m_removingTorrents.contains(infoHash)) {
         const RemovingTorrentData tmpRemovingTorrentData = m_removingTorrents[infoHash];
-        if (!tmpRemovingTorrentData.requestedFileDeletion) {
+        if (tmpRemovingTorrentData.deleteOption == Torrent) {
```

Comment:
This "if" isn't needed because it's always "Torrent" here but maybe it has some reason i don't know about?

Reply:
(Not sure) I think the whole `if (m_removingTorrents.contains(infoHash)) {}` isn't needed but the log msg should be preserved.
And this line should be removed too: https://github.com/qbittorrent/qBittorrent/blob/e741e353618a90915fc1352830787402193c9757/src/base/bittorrent/session.cpp#L1658

Reply:
>(Not sure) I think the whole if (m_removingTorrents.contains(infoHash)) {} isn't needed but the log msg below should be preserved.

We need the torrent name which is deleted by now for the msg.

Reply:
>We need the torrent name which is deleted by now for the msg.

I see. Then the `if ()` isn't necessary.